pkg/fleetautoscalers/fleetautoscalers.go

```diff
@@ -319,7 +319,7 @@ func applyCounterOrListPolicy(c *autoscalingv1.CounterPolicy, l *autoscalingv1.L
 		if err != nil {
 			return 0, false, err
 		}
-		// The desired TOTAL capacity based on the Aggregated Allocated Counts (see applyBufferPolicy for explanation)
+		// The desired AVAILABLE capacity based on the Aggregated Allocated Counts (see applyBufferPolicy for explanation)
```

Comment:
@igooch just triple checking - this needs changing too, since we're calculating the desired total capacity (making the change now too).

Reply:
Yep! Any priority related to fleet or fleet autoscaler is total capacity.